CHANGES

```diff
@@ -183,6 +183,8 @@ Standard Library
 - Added [Coq.Strings.String.concat] to concatenate a list of strings
   inserting a separator between each item
 - Notation `'` for Zpos in QArith was removed.
+- Added [Coq.Structures.OrderedTypeEx.String_as_OT] to make strings an
+  ordered type (using lexical order)
```

Comment:
This entry is out of place. It should be in the very top section (changes beyond 8.8) and there you can create a new "Standard Library" sub-section.